client/my-sites/current-site/index.jsx

```diff
@@ -156,11 +156,11 @@ module.exports = React.createClass( {
 		if ( ! this.props.sites.initialized ) {
 			return (
 				<Card className="current-site is-loading">
-					<div className="site">
 					{ hasOneSite
 						? this.addNewWordPressButton()
-						: <span className="current-site__switch-sites" />
+						: <span className="current-site__switch-sites">&nbsp;</span>
```

Comment:
Huh... I wasn't sure, disabled javascript during this state to inspect to see what happens, and it renders as a space.
![space](https://cloudup.com/c_EH1L92DLa+)